src/services/user-group/model.js

```diff
@@ -44,14 +44,18 @@ const courseSchema = getUserGroupSchema({
 	ltiToolIds: [{ type: Schema.Types.ObjectId, required: true, ref: 'ltiTool' }],
 	color: { type: String, required: true, default: '#ACACAC' },
 	startDate: { type: Date },
-	untilDate: { type: Date },
-	shareToken: { type: String, unique: true, sparse: true },
+	untilDate: { type: Date, index: true },
+	shareToken: { 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5221564)